src/config-schema.js

```diff
@@ -357,7 +357,7 @@ const configSchema = {
         description: 'Experimental: Use the new Tree-sitter parsing system for supported languages.'
       },
       colorProfile: {
-        description: "Specify whether Atom should use the operating system's color profile (recommended) or an alternative color profile.<br>Changing this setting will require a relaunch of Atom to take effect.",
+        description: "Specify whether Atom should use the operating system's color profile (recommended) or an alternative color profile.",
```

Comment:
The relaunch line is included in https://github.com/atom/atom/pull/17464/files#diff-8c99efd21425e5773e52ba4fbb820bf0L574, the other setting that requires a restart.  If this one is removed, then the other should be as well.

Reply:
Oh, I thought I checked that one, but guess I didn't scroll far enough. Good catch, @50Wliu :1st_place_medal: 
Personally I vote for removing it in both places. What do other folks think? /cc @atom/feedback

Reply:
I think it's good to inform the user that a restart will be required even if we're going to prompt them.  This will make sure they understand why they're being prompted for a restart.  I'd say leave the message in for the sake of clarity